pkg/reservations/base.go

```diff
@@ -53,12 +54,6 @@ type Reader interface {
 	ReadAll(io.Reader) ([]TZReservation, error)
 }
 
-type Writer interface {
```

Comment:
Let's talk about this one. I'd prefer to keep the interface here and have a JsonWriter and TextWriter implementation and a method to create the right implementation. If it's more concise, a functional approach is cool too.